client/order/index.js

```diff
@@ -1,30 +1,58 @@
 /* global jQuery */
 
-import { __, _n, sprintf } from '@wordpress/i18n';
-import { dateI18n } from '@wordpress/date';
 import ReactDOM from 'react-dom';
+import React from 'react';
+import { __ } from '@wordpress/i18n';
 import { dispatch } from '@wordpress/data';
-import moment from 'moment';
-import { createInterpolateElement } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { getConfig } from 'utils/order';
+import { isAwaitingResponse, isUnderReview } from 'wcpay/disputes/utils';
 import RefundConfirmationModal from './refund-confirm-modal';
 import CancelConfirmationModal from './cancel-confirm-modal';
-import InlineNotice from 'components/inline-notice';
-import { formatExplicitCurrency } from 'utils/currency';
-import { reasons } from 'wcpay/disputes/strings';
-import { getDetailsURL } from 'wcpay/components/details-link';
-import {
-	isAwaitingResponse,
-	isInquiry,
-	isUnderReview,
-} from 'wcpay/disputes/utils';
-import { useCharge } from 'wcpay/data';
-import wcpayTracks from 'tracks';
-import './style.scss';
+import DisputedOrderNoticeHandler from 'components/disputed-order-notice';
+
+function disableWooOrderRefundButton( disputeStatus ) {
+	const refundButton = document.querySelector( 'button.refund-items' );
+	if ( ! refundButton ) {
+		return;
+	}
+
+	refundButton.disabled = true;
+
+	// Show helpful info in order edit lock icon tooltip.
+
+	let tooltipText = '';
+	if ( isAwaitingResponse( disputeStatus ) ) {
+		tooltipText = __(
+			'Refunds and order editing are disabled during disputes.',
+			'woocommerce-payments'
+		);
+	} else if ( isUnderReview( disputeStatus ) ) {
+		tooltipText = __(
+			'Refunds and order editing are disabled during an active dispute.',
```

Comment:
Note I tweaked this wording to remove redundant "at this time". 

<img width="1096" alt="Screenshot 2023-10-30 at 2 33 24 PM" src="https://github.com/Automattic/woocommerce-payments/assets/4167300/37de4408-0573-4d15-aab2-5de08bf4b8ed">